exporters/utils.go

```diff
@@ -223,3 +227,33 @@ func RemoveElements(slice []string, drop []string) []string {
 	}
 	return res
 }
+
+func PrepareTLSConfig(caCertFile string) (*tls.Config, error) {
```

Comment:
does this function needs to be exported? can you add a unit test? (not blocker, just asking if you could)